src/__tests__/index.js

```diff
@@ -1,63 +1,86 @@
-import chai from 'chai';               // DOCUMENTATION: http://chaijs.com/
-import { mocks } from 'mock-browser';  // DOCUMENTATION: https://www.npmjs.com/package/mock-browser
+/* globals jest describe test expect */
 
-const { describe, it } = global;
+import analytics from '../../examples/react-router/imports/analytics.min';
+import OKGAnalytics, { trackPageWhenReady, trackEventWhenReady, identifyWhenReady } from '../index';
 
-// Let's setup the global browser objects our, and the underlying analytics.js, package requires...
-const mock = new mocks.MockBrowser();
-
-global.window = mock.getWindow();
-global.document = mock.getDocument();
-global.location = mock.getLocation();
-global.navigator = mock.getNavigator();
-global.history = mock.getHistory();
-
-// Now that's all setup, pull in the package...
-const OKGAnalytics = require('../index');
-
-const should = chai.should();
+// Params to pass to initialize OKGAnalytics
+const validSettings = { // eslint-disable-line
+  analytics,
+  integrations: {
+    'Google Analytics': { 'trackingId': 'UA-58359748-3' }, // eslint-disable-line quote-props
+    'Mixpanel': { 'token': 'b513b13a2e253576934b47d2a195ae29', 'people': true }, // eslint-disable-line quote-props
+  },
+  options: {
+    // Segment options for initialize() from analytics.js-core
+  },
+  autorun: true,
+};
 
 // Some very basic "smoke test" stuff...
-describe('okgrow-auto-analytics imports', () => {
-  it('analytics is an object', async () => {
-    OKGAnalytics.analytics.should.be.a('object');
+describe('@okgrow/auto-analytics correctly exposes', () => {
+  test('trackEventWhenReady is a function', () => {
+    expect(trackEventWhenReady).toBeDefined();
```

Comment:
I suppose we _could_ add something like `expect(typeof trackEventWhenReady).toBe('function')` But that might be overkill.

Reply:
@ccuilla I like the idea! Updated to also check the `type`. 🔥